pkg/frontend/nvme_subsystem_test.go

```diff
@@ -309,9 +309,7 @@ func TestFrontEnd_UpdateNvmeSubsystem(t *testing.T) {
 	}{
 		"invalid fieldmask": {
 			&fieldmaskpb.FieldMask{Paths: []string{"*", "author"}},
-			&pb.NvmeSubsystem{
-				Name: testSubsystemName,
```

Comment:
not sure this is correct... in update we need Name... which is missing from `testSubsystem`

Reply:
Added as a comment to this issue:
https://github.com/opiproject/opi-spdk-bridge/issues/443